aptos/typescript/wallet_adapter_react/src/server/main.ts

```diff
@@ -0,0 +1,199 @@
+import express from "express";
+import ViteExpress from "vite-express";
+import { 
+  GasStationClient, 
+  WalletClient,
+  ShinamiWalletSigner,
+  KeyClient
+} from "@shinami/clients/aptos";
+import dotenvFlow from 'dotenv-flow';
+
+import { 
+  Aptos, 
+  AptosConfig,
+  Network, 
+  AccountAddress, 
+  SimpleTransaction,
+  MoveString,
+  Deserializer,
+  Hex,
+  AccountAuthenticator
+} from "@aptos-labs/ts-sdk";
+
+// Get our environmental variables from our .env.local file
+dotenvFlow.config();
+export const ALL_SERVICES_TESTNET_ACCESS_KEY = process.env.ALL_SERVICES_TESTNET_ACCESS_KEY;
+export const USER123_WALLET_SECRET = process.env.USER123_WALLET_SECRET;
+export const USER123_WALLET_ID = process.env.USER123_WALLET_ID;
+
+if (!(ALL_SERVICES_TESTNET_ACCESS_KEY)) {
+  throw Error('ALL_SERVICES_TESTNET_ACCESS_KEY .env.local variable not set');
+}
+if (!(USER123_WALLET_ID && USER123_WALLET_SECRET)){
+  throw Error('USER123_WALLET_ID and/or USER123_WALLET_SECRET .env.local varaibles not set');
+}
+
+// Create an Aptos client for building and submitting transactions.
+const aptosClient = new Aptos(new AptosConfig({ network: Network.TESTNET}));
+
+// Create Shinami clients for sponsoring transactions and for our Invisible Wallet operations.
+const gasClient = new GasStationClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const keyClient = new KeyClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const walletClient = new WalletClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+
+// Create our Invisible Wallet 
+const signer = new ShinamiWalletSigner(
+  USER123_WALLET_ID,
+  walletClient,
+  USER123_WALLET_SECRET,
+  keyClient
+);
+const CREATE_WALLET_IF_NOT_FOUND = true;
+const WALLET_ONE_SUI_ADDRESS = await signer.getAddress(CREATE_WALLET_IF_NOT_FOUND);
+console.log("Invisible wallet address:", WALLET_ONE_SUI_ADDRESS.toString());
+
+// initilaize our server
+const app = express();
+app.use(express.json());
+
+ViteExpress.listen(app, 3000, () =>
+  console.log("Server is listening on port 3000..."),
+);
+
+
+
+// Endpoint to:
+//  1. Build and a feePayer SimpleTransaction with the given user intput.
+//  2. Sign, sponsor, and submit it for an Invisible Wallet sender.
+//  3. Return the PendingTransactionResponse to the FE
+app.post('/invisibleWalletTx', async (req, res, next) => {
```

Comment:
Please add a callout that all of these APIs are UNAUTHENTICATED, and they must add their own user auth / session management to secure them.

Reply:
Done.

Reply:
done